packages/react-router/src/context.ts

```diff
@@ -1,8 +1,14 @@
-import {createContext} from 'react';
+import React, {createContext} from 'react';
 import {EnhancedURL} from './types';
 
 export const CurrentUrlContext = createContext<EnhancedURL | null>(null);
 export const RouterContext = createContext<import('./router').Router | null>(
   null,
 );
 export const SwitchContext = createContext<{matched(): void} | null>(null);
+
+export interface FocusRef {
+  focusRef: React.RefObject<HTMLElement>;
```

Comment:
Use the named exports for React instead of accessing values off the "default" export